README.md

```diff
@@ -1,30 +1,60 @@
-The next option is to select a mod and build the project according to it.
-And lastly launching the project. Also these possibilities should be available for a group of projects, i.e. the selection of settings and then the execution of the necessary logic for all of them.
+The tool should be able to:
+
+1. Synchronize repositories (all or selected). Pull the branch that is currently selected. Selected branch shall be visible to user.
+2. Have an autostash feature - be able to stash changes for current branch before pulling.
+3. Have a yaml configuration file in a repo to get the configuration from. App configuration should contains paths to the local repositories.
+4. Automatically prune branches that do not have remote branches.
+5. Have the ability to manually add repositories for synchronization from FossConnect project.
+6. Have some short meaningfull output in the UI and detailed logs in a file. There should be a dedicated buttons: 'View logs' to open log folder for convinience, 'View config' to open folder with it.
+7. Build selected projects.
+8. Run selected projects.
+9. Introduce profiles that will contain list of repositories that should be checked for synchronization.
+10. Have the ability to start the tool from the command line passing the profile name as a parameter.
 
 ### Implementation
 
-This will be a desktop application developed using WPF. The configuration file mentioned earlier will be local in the first iteration, in the future it is planned to load it from a remote repository. 
-It is planned to use the git application to work with repositories because the pool of scheduled tasks does not require complex commands. List of commands that are planned to be used are : branch, checkout, pull, stash. 
-All information will be presented in a table. One line in it corresponds to one project. 
-To select the necessary branch there will be a ComboBox with an available selection. After branch selection, SourceExplorer will check if the branch is presented locally. If not - it will perform a pull operation. 
-Also, there will be a ComboBox for build mode.
-For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button.
+This will be a desktop application developed using WPF targeting .NET 6. It is planned to use the libgit2shart nuget package to work with repositories because this library is time-tested and will allow you to perform all planned operations and expand their list in the future. List of commands that are planned to be used now are : branch, checkout, pull, stash.
+
+All information will be presented in the table. One line in it corresponds to one project.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/mainwindow.png)
+
```

Comment:
still top margin is too small
![image](https://user-images.githubusercontent.com/42497175/143389435-8fa33d59-1904-4a41-aee9-b874d28e0360.png)

Reply:
fixed